src/components/LessonInfoDisplay/LessonInfoDisplay.scss

```diff
@@ -79,7 +82,14 @@
   transition-duration: 0.3s;
 }
 
-.CloseButton:hover {
+.CloseButton2:hover {
   background-color: #ffc700;
   color: white;
 }
+
+.buttonContainer {
```

Comment:
please name your classes consistently- don't have them be camelcase and then have the beginning lettter uppercase etc